src/wp-includes/rest-api/endpoints/class-wp-rest-block-navigation-areas-controller.php

```diff
@@ -0,0 +1,285 @@
+<?php
+/**
+ * REST API: WP_REST_Block_Navigation_Areas_Controller class
+ *
+ * @subpackage REST_API
+ * @package    WordPress
```

Comment:
```suggestion
 * @package WordPress
```

These tags don't get aligned, https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#6-file-headers.